source/_components/googlehome.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "Google Home"
+description: "Instructions on how to connect to your Google Home device."
+date: 2018-11-04 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google_home.png
+ha_release: 0.88
+ha_category:
+  - Hub
+  - Presence Detection
+ha_iot_class: "Local Polling"
+redirect_from:
+  - /components/device_tracker.googlehome/
+---
+
+The `googlehome` component allows you to connect to your Google Home device using an [unofficial Google Home API][googlehomeapi].
+
+This component will set up a [device_tracker](/components/device_tracker/) platform to track nearby bluetooth devices.
+
+## {% linkable_title Configuration %}
+
+To integrate the `googlehome` component in Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+googlehome:
+  devices:
+    - host: IP_ADDRESS
+```
+
+{% configuration %}
+devices:
+  description: A list of Google Home devices.
+  required: true
+  type: list
+  keys:
+    host:
+      description: The IP address of the Google Home device.
+      required: true
+      type: string
+    rssi_threshold:
+      description: The threshold for the RSSI signal of the device.
+      required: false
+      default: -70
+      type: integer
+    device_types:
+      description: Device types that will be tracked [see device types](#device_types), by default all types are tracked.
+      required: false
+      type: list
+      keys:
+        1:
```

Comment:
Maybe this just looks messy? If we want to keep it we have to quote the numbers. But I think we can revert to the previous way it was. Sorry.

Reply:
This way is more correct according to the standard, but I agree that this looks messier 🤷‍♂️

Reply:
Added quotes now, so we can see when it has rendered :)

Reply:
The previous way was better I think. I don't understand what 1, 2, 3 means.